WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -374,10 +374,13 @@ private extension WooShippingCreateLabelsView {
             Text(Localization.BottomSheet.shipFrom)
                 .trackSize(size: $shipmentDetailsShipFromSize)
 
-            if viewModel.canViewLabel,
-               let addressLines = viewModel.originAddressLines {
-                AddressLinesView(addressLines: addressLines)
-                    .frame(maxWidth: .infinity, alignment: .leading)
+            if viewModel.canViewLabel {
+                AddressLinesView(
+                    addressLines: viewModel.originAddressLines ??
+                    /// Fallback to `originAddress` in case if `originAddressLines` are missing
+                    [viewModel.originAddress]
```

Comment:
I'm concerned that this fallback may cause the app to display incorrect data. Do you think we can remove the workaround if we add support for the origin address array as suggested above?

Reply:
I'm good with having a blank space, it's easier to know that there's an issue than having incorrect data.